package.json

```diff
@@ -36,6 +36,7 @@
   "dependencies": {
     "appium-android-driver": "1.x",
     "appium-base-driver": "2.x",
+    "appium-espresso-driver": "^1.0.0-beta.1",
```

Comment:
Not sure. I'll have to look into that. I would like it if it always was the latest beta.